release-notes.md

```diff
@@ -1,3 +1,47 @@
+## 2018-05-01
+
+
+## Install
+
+To install decrediton download, uncompress, and run
+[decrediton Linux](https://github.com/decred/decred-binaries/releases/download/v1.2.1/decrediton-1.2.1.tar.gz) or
+[decrediton OSX](https://github.com/decred/decred-binaries/releases/download/v1.2.1/decrediton-1.2.1.dmg) or
+[decrediton Windows](https://github.com/decred/decred-binaries/releases/download/v1.2.1/decrediton-1.2.1.exe).
+
+See  manifest-decrediton-v1.2.1.txt, and the package specific manifest files for sha256 sums and the associated .asc files to confirm those shas.
+
+See [README.md](./README.md#verifying-binaries) for more info on verifying the files.
+
+
+# decrediton v1.2.1
+
+This decrediton patch release covers a few commonly reported issues shortly
+after v1.2.1 was released.  Most importantly this fixes an issue causing 
```

Comment:
This decrediton patch releases covers a few issues that were commonly reported shortly after v1.2.0 was released.